docs/how-tos/respond-to-returned-request.md

```diff
@@ -18,7 +18,7 @@ file and two files with changes requested:
 ## Comments
 
 Navigate to each of your file groups to view comments from the output checkers. You should add
-comments to respond to any questions or requests for information. Note that your comments will
+comments [formatted as needed](../reference/formatting-comments.md), to respond to any questions or requests for information. Note that your comments will
 not be visible to output checkers until you have submitted the request again for review
 
```

Comment:
I'd suggest adding a separate line about formatting, with the link, so it's more obvious e.g.

"You can [write your comments in markdown](...) for additional formatting."